Source/ALSV4_CPP/Public/Character/ALSCharacterMovementComponent.h

```diff
@@ -51,23 +51,29 @@ class ALSV4_CPP_API UALSCharacterMovementComponent : public UCharacterMovementCo
 	void OnMovementUpdated(float DeltaTime, const FVector& OldLocation, const FVector& OldVelocity);
 
 
-	// Set Max Walk Speed
-	uint8 bRequestMaxWalkSpeedChange : 1;
+	// Movement Settings Variables
+	uint8 bRequestMovementSettingsChange : 1;
 
 	float MyNewBraking;
 	float MyNewGroundFriction;
 	float MyNewMaxWalkSpeed;
 	float MyNewMaxAcceleration;
 
-	// Set Max Braking Deceleration Walking and Ground Friction
+	float RealBraking;
+	float RealGroundFriction;
+	float RealMaxAcceleration;
 
+	// Set Max Walking Speed (Called from the owning client)
 	UFUNCTION(BlueprintCallable, Category = "Movement Settings")
-	void SetBrakingAndGroundFriction(float NewBraking, float NewGroundFriction);
+		void SetMaxWalkingSpeed(float NewMaxWalkSpeed);
 
-	// Set Max Walk Speed (Called from the owning client)
+	UFUNCTION(reliable, Server, WithValidation)
+		void Server_SetMaxWalkingSpeed(const float NewMaxWalkSpeed);
```

Comment:
Can you fix indentations here and line 68?

Reply:
Done